webapp/src/utils.ts

```diff
@@ -77,6 +77,15 @@ export function getChannelURL(state: GlobalState, channel: Channel, teamId: stri
     return channelURL;
 }
 
+export function shouldRenderCallsIncoming() {
+    const win = window.opener ? window.opener : window;
+    if (win.desktop && window.location.pathname.indexOf('/messages/') === -1) {
```

Comment:
Good question.
I was kind of assuming `/messages` wouldn't change... But it might... And anyway, I discovered this was wrong (you could break it under some combinations of reloading and where you were in the app).  So I've changed the `index.tsx` and the `call_container` to fix that, and I changed this util fn to do the check a bit more safely. Hopefully this will be a bit more robust for when we remove the tabs: there's a good chance that when the tabs are removed we will need to display the LHS incoming widget no matter what you're looking at, and this should work then. 
(Remember, the reason we're doing this check is that we don't want multiple LHS incoming containers to ring at the same time, which is what was happening because the desktop loaded separate webapps per tab.)